static/src/javascripts/bootstraps/commercial.js

```diff
@@ -89,7 +89,8 @@ define([
             ['cm-hostedOJCarousel', hostedOJCarousel.init]);
     }
 
-    if (!config.page.disableStickyTopBanner) {
+    var supportsSticky = document.documentElement.classList.contains('has-sticky');
+    if (!config.page.disableStickyTopBanner && !supportsSticky) {
```

Comment:
Yes I do 😲  I don't know how we'd do without them though, except if one day we draw a line in the sand and stop making the banner stick on those browsers that don't support `position: sticky`. But, to match the way other modules work, I could just add a flag in `commercialFeatures`. Let's do that!